### PR TITLE
Add OpenAssistant and HuggingChat provider, Remove HugChat

### PR DESCRIPTION
**Pull request by hlohaus**

### CODE REVIEW

**Comment by xtekky** [comment]
looks like there is a few conflicts

**Comment by hlohaus** [comment]
I pulled the master

File: g4f/Provider/Liaobots.py [hunk 1]
```diff
@@ -1,59 +1,77 @@
-import uuid, requests
+import uuid
+import json
+from aiohttp import ClientSession
 
-from ..typing       import Any, CreateResult
-from .base_provider import BaseProvider
+from ..typing import AsyncGenerator
+from .base_provider import AsyncGeneratorProvider
```
Comment (monjie110) [inline g4f/Provider/Liaobots.py:6]: سلام

**Review by monjie110 — APPROVED** [review]
شاداب باشید

**Pull request merged**